src/Illuminate/Database/Eloquent/Builder.php

```diff
@@ -929,6 +941,14 @@ public function __call($method, $parameters)
 		{
 			return $this->callScope($scope, $parameters);
 		}
+		elseif ( ! is_null($this->model['fillable']) && in_array($column = 'is_'.$method, $this->model['fillable']))
+		{
+			return $this->addBooleanColumnScope($column, true);
+		}
+		elseif ( ! is_null($this->model['fillable']) && in_array($column = 'is_'.lcfirst(str_replace('not', '', $method)), $this->model['fillable']))
```

Comment:
What happen if you have `is_annotation` attribute and you attempt to use `Model::notAnnotation()`. Wouldn't this look for `is_anation` due to `str_replace`?